### PR TITLE
Add tag support

### PR DESCRIPTION
**Pull request by jcogilvie**
It's not the prettiest, but it does the job.  

image

image

image

image

### CODE REVIEW

**Comment by jcogilvie** [comment]
Should fix #70.

**Comment by jcogilvie** [comment]
Updated favorite stars.  

image

File: ScratchEVTCParser.sln [hunk 1]
```diff
@@ -1,28 +1,31 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EVTCAnalytics", "EVTCAnalytics\EVTCAnalytics.csproj", "{9DD47A76-3C69-4152-AA36-B656CAFFDECB}"
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30517.126
```
Comment (Sejsel) [inline ScratchEVTCParser.sln:4]: Why, thank you, Visual Studio, for unnecessary merge conflicts :)

**Review by Sejsel — COMMENTED** [review]
This looks pretty good to me, there are a few tiny things that I think could be improved in individual comments. If you don't have time to spend on it, I can implement them myself, just let me know. Big thanks for the nice work in any case!

**Review by Sejsel — COMMENTED** [review]
Oh, and it would be also nice to run automatic formatting on the files, I saw some errant formatting and trailing spaces in places (Ctrl K + D in Visual Studio).

**Pull request merged**